README.md

```diff
@@ -73,56 +82,75 @@ eclipse
 
-### Step 5: Import launch configuration.
+### Step 5: Configure the Toolchain
+This step configures Eclipse so that it finds the CORE-V toolchain that was downloaded and installed in **Step 1**.
+
+In the Eclipse Project Explorer select the `cli_test` Project, right click and select `Properties` (typically appears at the bottom of the pop-up menu).
+Select `C/C++ Build / Settings` to bring up the window below:
+
+![Step 5.1](./images/5p1-Properties_for_cli_test.png)
+
+In this example, the `Toolchain path` was changed by selecting the `workspace` preference page.
+This brings up the "Workspace RISC-V Toolchains Paths" preferences selection window.
+Update the Toolchain folder to point to your install path.
+
+![Step 5.2](./images/5p2-WorkspaceRISC-VToolchainsPaths.png)
+
+### Step 6: Compile cli_test
+Assuming everything above has completed this step is simplicity itself.
+With `cli_test` selected in the Project Explorer pane, click on Build (the hammer icon):
+
+![Step6.0](./images/BuildCliTest.png)
+
+### Step 7: Import launch configuration
+
+This step pulls in the launch configurations which are configured to use **OpenOCD** and the [JTAG-HS2](https://digilent.com/shop/jtag-hs2-programming-cable/) programmer.
+Reminder: the instructions for connecting the HS2 to the Nexys A7 can be found in the CORE-V-MCU Quick Start Guide (a link can be found at the top of this README).
 
-This step pulls in the launch configurations which are configured to use hs2 and OpenOCD.
 If your hardware setup is different, you can either ignore this step and create your own, or use this step and modify to fit your configuration.
 
-#### Step 5.1:
-Select 'Import...'.
+#### Step 7.1:
+Select 'File' and then 'Import...'.
 
 ![step11](./images/11-ImportLaunch.png)
 
-#### Step 5.2:
-Select Launch Configurations.
+#### Step 7.2:
+Select 'Run/Debug' and then 'Launch Configurations'.
 
 ![step12](./images/12-SelectLaunchConfig.png)
 
-#### Step 5.3:
-Browse to the git directory that was chosen as the Local Destination.
-
-![step13](./images/13-ImportLaunchConfigurations.png)
+#### Step 7.3:
+Here you will fetch the HS2 launch configuration from your clone of core-v-mcu-cli-test repository.
+In the `core-v-mcu-cli-test` repo there is a set of launch configurations in the `launch` directory.
+Use the `Browse...` button to navigate to the git directory that was chosen as the 'Local Destination' in **Step 4.7** and then `launch`.
 
-#### Step 5.4:
-Select launchconfig and cli_test Default.
+Select 'launch' and 'cli_test hs2.lauch'.
 
 ![step14](./images/14-SelectLaunchConfigurations.png)
 
-#### Step 5.5:
+#### Step 7.5:
 Under the `Run` menu select `Debug Configurations...`
 
 ![step15](./images/15-DebugConfig.png)
 
-#### Step 5.6:
+#### Step 7.6:
 Under `GDB OpenOCD Debugging` select `cli_test Default` and then `Debug`.
 
 ![step16](./images/16-cli_testDefault.png)
 
 This should compile, link and load the application and stop at main waiting for you to start debugging.
```

Comment:
the debug session runs, but on the minicom uart port I get 'A2 BOOTME' scrolling with <CR> instead of the 0> cli prompt

![Screenshot from 2022-03-12 19-25-35](https://user-images.githubusercontent.com/2498391/158039667-d6f7581f-5590-4fd5-a612-de083205a661.png)

Reply:
That indicates that the emulated MCU is loaded in the FPGA and running its bootloader.  If there were no error messages from Eclipse then cli_test _should_ have been loaded into memory via the JTAG.  What was the message you got from Eclipse when running this step?

Reply:
@rickoco This may be due to the extra reset in the debug configuration.  -- edit the debug configurations and in the  startup tab make sure the Pre-run/Restart reset is *not* checked.  This seems to generate a reset after downloading that then traps back into the boot loader

Reply:
Thx @gmartin102 - that was the issue.

@MikeOpenHWGroup We need to insert an additional instruction to make sure Run/Restart settings are correct.  I've attached an image with the correct settings and we can add text such as: "Edit the debug configurations and in the Startup tab scroll down to the Run/Restart Commands section to make sure the Pre-run/Restart Reset box is not checked."
![Screenshot from 2022-03-14 17-42-04](https://user-images.githubusercontent.com/2498391/158266862-458db73b-5aa8-4ebe-ba3f-ebddba99809c.png)

Reply:
Done!  In fact, I stole yours.  Thanks.